### PR TITLE
Clean up and re-org

### PR DESCRIPTION
**Pull request by mosharaf**

### CODE REVIEW

File: _includes/topnav.html [hunk 1]
```diff
@@ -14,98 +14,49 @@
...
-                    <ul class="dropdown-menu">
-                        <li>
-                            <a href=https://fedscale.readthedocs.io/en/latest>Overview</a>
-                        </li>
-                        <li class="divider"></li>
-                        <li>
-                            <a href=https://fedscale.readthedocs.io/en/latest/fedscale.core.aggregation.html#module-fedscale.core.aggregation.aggregator>Aggregator</a>
+                            <a href={{ "/leaderboard/aggregation.html"  | relative_url }}>Aggregation Optimizations</a>
                         </li>
-                        <li class="divider"></li>
```
Comment (fanlai0990) [inline _includes/topnav.html:75]: Are you removing these "divider"s intentionally? It does not look good to me, as texts are crowded.

Comment (mosharaf) [reply _includes/topnav.html:75]: Dividers between every item has little value. There are dividers between groups of related items

**Comment by fanlai0990** [comment]
Thanks a lot for cleaninng up these! Btw, are we considering #4?

**Comment by mosharaf** [comment]
No :|

**Pull request merged**